addons/purchase_requisition/views/purchase_views.xml

```diff
@@ -28,7 +28,7 @@
                             </p>
                         </group>
                     </group>
-                    <field name="alternative_po_ids" attrs="{'readonly': [('id', '=', False)]}" widget="many2many_alt_pos" context="{'quotation_only': True}">
+                    <field name="alternative_po_ids" widget="many2many_alt_pos" context="{'quotation_only': True}">
```

Comment:
It's unlikely that you will have any alternative POs linked without saving the record first... this change doesn't seem useful